### ISSUE TITLE
Build fail for taichi legacy branch on ubuntu 20.04

### ISSUE THREAD

**Issue by ghost** [taichi-dev/taichi#2311]
I am willing to work with code [spgrid_topo_opt](https://github.com/yuanming-hu/spgrid_topo_opt) which needs [taichi legacy](https://github.com/yuanming-hu/taichi/tree/legacy) to be installed first. 

To install taichi legacy following the instruction on this [page](https://taichi.readthedocs.io/en/stable/legacy_installation.html) I run the command
`wget https://raw.githubusercontent.com/yuanming-hu/taichi/legacy/install.py` 
then `python3.8 install.py` which gives me the following error message : 

`make[2]: *** [CMakeFiles/taichi_core.dir/build.make:193: CMakeFiles/taichi_core.dir/include/taichi/math/svd.cpp.o] Error 1 make[1]: *** [CMakeFiles/Makefile2:96: CMakeFiles/taichi_core.dir/all] Error 2 make: *** [Makefile:84: all] Error 2 Error: Build failed. Error: installation failed.`

The code above is actually the very last fraction of the output, I don't seem to find the log file of the whole output to share it with you here if there is a way to have the log file containing the whole information of building process let me know please.

The bottom line is I tried couple of different ways to download `install.py` and install taichi legacy with it but every time I get the same error message.

What am I missing here? Anybody here ever tried to install taichi legacy?

By the way I also ran `ti diagnose` and here is the message:

`make[1]: *** [CMakeFiles/Makefile2:96: CMakeFiles/taichi_core.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
  Error: Build failed.`

Again this is only the last couple of lines of the output. How can I find the whole output file so I could share it with you?
My machine is running on ubuntu 20.04.

Thanks a bunch

**Comment by rexwangcc** [comment]
Hi, sorry for the delayed reply, we have moved the issue to our tracking board which will be worked on later, meanwhile, if you could, we highly recommend switching to the latest version of Taichi from the legacy code base for better compatibility and support, thanks!!

**Comment by ailzhang** [comment]
Closing via https://github.com/taichi-dev/taichi/issues/2311#issuecomment-1038736359

**Comment by Shatom** [comment]
Have you solved that? I met the same problem...

**Comment by Shatom** [comment]
> I am willing to work with code [spgrid_topo_opt](https://github.com/yuanming-hu/spgrid_topo_opt) which needs [taichi legacy](https://github.com/yuanming-hu/taichi/tree/legacy) to be installed first.
> 
> To install taichi legacy following the instruction on this [page](https://taichi.readthedocs.io/en/stable/legacy_installation.html) I run the command `wget https://raw.githubusercontent.com/yuanming-hu/taichi/legacy/install.py` then `python3.8 install.py` which gives me the following error message :
> 
> `make[2]: *** [CMakeFiles/taichi_core.dir/build.make:193: CMakeFiles/taichi_core.dir/include/taichi/math/svd.cpp.o] Error 1 make[1]: *** [CMakeFiles/Makefile2:96: CMakeFiles/taichi_core.dir/all] Error 2 make: *** [Makefile:84: all] Error 2 Error: Build failed. Error: installation failed.`
> 
> The code above is actually the very last fraction of the output, I don't seem to find the log file of the whole output to share it with you here if there is a way to have the log file containing the whole information of building process let me know please.
> 
> The bottom line is I tried couple of different ways to download `install.py` and install taichi legacy with it but every time I get the same error message.
> 
> What am I missing here? Anybody here ever tried to install taichi legacy?
> 
> By the way I also ran `ti diagnose` and here is the message:
> 
> `make[1]: *** [CMakeFiles/Makefile2:96: CMakeFiles/taichi_core.dir/all] Error 2 make: *** [Makefile:84: all] Error 2 Error: Build failed.`
> 
> Again this is only the last couple of lines of the output. How can I find the whole output file so I could share it with you? My machine is running on ubuntu 20.04.
> 
> Thanks a bunch

Did you use WSL? Someone said there is no problem in ubuntu, but WSL...

**Comment by automatingdesign22** [comment]
The same problem. Have you solved it?
